### PR TITLE
Alice pubs, talks, posters

### PR DESCRIPTION
**Pull request by AliceQingyangWang**

### CODE REVIEW

File: content/bibs/pubs.bib [hunk 1]
```diff
@@ -4026,3 +4026,26 @@ @article{athey2023brainline
...
+  publisher = {PMLR},
+  year={2023},
+  abstract={Natural intelligences (NIs) thrive in a dynamic world - they learn quickly, sometimes with only a few samples. In contrast, artificial intelligences (AIs) typically learn with a prohibitive number of training samples and computational power. What design principle difference between NI and AI could contribute to such a discrepancy? Here, we investigate the role of weight polarity: development processes initialize NIs with advantageous polarity configurations; as NIs grow and learn, synapse magnitudes update, yet polarities are largely kept unchanged. We demonstrate with simulation and image classification tasks that if weight polarities are adequately set a priori, then networks learn with less time and data. We also explicitly illustrate situations in which a priori setting the weight polarities is disadvantageous for networks. Our work illustrates the value of weight polarities from the perspective of statistical and computational efficiency during learning.},
+  arxiv={2303.17589},
+  author+an = {1=trainee;2=trainee;3=trainee;4=trainee;5=highlight},
+  }
+
+@InProceedings{Wang_2023_ICCV,
```
Comment (SUKI-O) [inline content/bibs/pubs.bib]: can you please fix the typo "InProceedings" --> "inproceedings"

**Review by SUKI-O — COMMENTED** [review]
The build failed in CI so there is something wrong with the added code. Can you fix the typo I noted and others if you find? Also, can you please run a formatting? I think it uses `black`.

**Review by SUKI-O — COMMENTED** [review]
Can you also change the name entry to `family name, given name` format? Thx :)

**Comment by AliceQingyangWang** [comment]
All conflicts resolved, checks passed, ready to be merged

**Review by SUKI-O — APPROVED** [review]

**Pull request merged**